src/main/kotlin/com/terraformation/backend/accelerator/api/CohortsController.kt

```diff
@@ -0,0 +1,115 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.accelerator.db.CohortStore
+import com.terraformation.backend.accelerator.model.ExistingCohortModel
+import com.terraformation.backend.accelerator.model.NewCohortModel
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.SimpleSuccessResponsePayload
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.db.default_schema.CohortId
+import com.terraformation.backend.db.default_schema.CohortPhase
+import com.terraformation.backend.db.default_schema.ParticipantId
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/cohorts")
+@RestController
+class CohortsController(private val cohortStore: CohortStore) {
+  @ApiResponse(responseCode = "200")
+  @ApiResponse404
+  @GetMapping("/{cohortId}")
+  @Operation(summary = "Gets information about a single cohort.")
+  fun getCohort(@PathVariable("cohortId") cohortId: CohortId): CohortResponsePayload {
+    val cohort = cohortStore.fetchOneById(cohortId)
+    return CohortResponsePayload(CohortPayload(cohort))
+  }
+
+  @ApiResponse(responseCode = "200")
+  @GetMapping()
+  @Operation(summary = "Gets the list of cohorts.")
+  fun getCohortList(): CohortListResponsePayload {
```

Comment:
I've been using a controller method naming convention of `getX` for the single-item fetch endpoints and `listX` for the listing ones. Not really important and it was an arbitrary choice on my part but may as well be consistent.